media-gfx/librecad/librecad-9999.ebuild

```diff
@@ -32,25 +32,54 @@ RDEPEND="
 	media-libs/freetype:2
 "
 DEPEND="${RDEPEND}
+	dev-qt/linguist-tools:5
```

Comment:
BDEPEND, and it's already there. It's just because of the PATH issue in the bug.

Reply:
fixed, thanks for heads up and fast review...